core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -59,7 +59,8 @@ object RuntimeConfigRequest {
                                   numberOfWorkerLocalSSDs: Option[Int] = None, //min 0 max 8
                                   numberOfPreemptibleWorkers: Option[Int] = None,
                                   properties: Map[String, String],
-                                  region: Option[RegionName] = None)
+                                  region: Option[RegionName],
+                                  componentGatewayEnabled: Option[Boolean])
```

Comment:
should it just be `Boolean`? just so we need one less layer of default? (default to `None`, and when it's `None`, default to `false`...so ultimately the default is `false` unless user says `true`

Reply:
Yeah that makes sense -- I think I made it `Option` here because it's optional in the request json (for backwards compatibility). But we can achieve that in the decoder too and make the class field `Boolean`.